src/signals/incident-management/containers/IncidentDetail/components/Detail/index.js

```diff
@@ -80,6 +80,10 @@ const Detail = ({ attachments, context }) => {
       ),
     [context?.near?.signal_count]
   )
+  const showPhone = useMemo(
+    () => /^\d+$/.test(incident.reporter.phone),
```

Comment:
This will not work. Valid phone number values are also `+312012345678`,  `020-12345678` and `020 12345678`

Reply:
Herjen said that only numbers could be present. Perhaps I should have tested that...